WickedEngine/wiGraphicsDevice.h

```diff
@@ -44,6 +44,12 @@ namespace wi::graphics
 
 		QUEUE_COUNT,
 	};
+	enum BACKEND_TYPE
```

Comment:
I would prefer to use dynamic_cast instead and not have this enum.

Reply:
Not sure , you would have to use the enum anyway , or what do you mean ?

Reply:
Ahh know what you mean now, fixed :)